helm/blueapi/.schema.yaml

```diff
@@ -0,0 +1,22 @@
+# .schema.yaml
```

Comment:
Which of these fields are required? Can we get away with using the defaults? e.g. noAdditionalProperties- why false but then additionalProperties: false in schema root? And why id of example.com?